app/views.py

```diff
@@ -72,7 +72,7 @@ def create_deck():
 
         deck = DecksRepository.create(user_id, name)
 
-        if DecksRepository.save(deck):
+        if DecksRepository.create(deck):
```

Comment:
whats that double create (73 and 75 lines)

Reply:
done